style.css

```diff
@@ -1,4 +1,961 @@
 body {
-  margin: 0;
-  padding: 0;
+  overflow-x: hidden;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+img {
+  border: 0;
+}
+
+/* .wrap {
+  width: 100%;
+} */
+
+p {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
```

Comment:
- [ ] Please, consider that the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop. Here I can see a few lines for the mobile version and then the whole CSS file is with @media